.github/workflows/jupyter.yaml

```diff
@@ -0,0 +1,58 @@
+name: "PyVAST Python Egg"
```

Comment:
```suggestion
name: "Jupyter Notebook"
```

Can you add a badge to the README so we have the workflow status at a glance for master?

Reply:
This is not yet resolved @0ortmann.